src/pages/waiting-room-to-car/cat-be/waiting-room-to-car.cat-be.page.ts

```diff
@@ -192,31 +181,6 @@ export class WaitingRoomToCarCatBEPage extends BasePageComponent {
         select(getVehicleDetails),
         map(isManual),
       ),
-      tellMeQuestionSelected$: currentTest$.pipe(
```

Comment:
What would be the reason for deleting these observables from the PageState? The html template uses them

Reply:
these were old observables from the cat b, so no longer needed.